locales/en.js

```diff
@@ -11,5 +11,9 @@
 	"EmailExist": "email already taken",
 	"localLoginSuccess": "Successfully logged in",
 	"internalServerError": "This service is currently unavailable. Developers at barefoot nomad are maintaining it",
-	"outSuccessfully": "Logout successful"
-}
+	"outSuccessfully": "Logout successful",
+	"emailSent": "We have sent you an email",
+	"passwordReset": "Password reset succesfully",
+	"password": "password",
+	"passwordsDontMatch": "Passwords do not match"
+}
```

Comment:
```suggestion
}

```
please add a new line